### DISCUSSION TITLE
Windows 11 path not found

### DISCUSSION THREAD

**Discussion by master-key-g** [huggingface/diffusers#11482] [Q&A]
Ok my proplem is this i try to train with train_dreambooth_lora_sdxl.py. I am running it like this.
accelerate launch train_dreambooth_lora_sdxl.py --pretrained_model_name_or_path=$MODEL_NAME --instance_data_dir=$INSTANCE_DIR --pretrained_vae_model_name_or_path=$VAE_PATH --output_dir=$OUTPUT_DIR --mixed_precision="fp16" --instance_prompt="ther is text to" --resolution=1024 --train_batch_size=1 --gradient_accumulation_steps=4 --learning_rate=1e-4 --lr_scheduler="constant" --lr_warmup_steps=0 --max_train_steps=500 --validation_prompt="ther is text" --validation_epochs=25 --seed="0" --enable_xformers_memory_efficient_attention --gradient_checkpointing --use_8bit_adam
The env's are set like this 
$env:INSTANCE_DIR="path to the folder with the images"
$env:VAE_PATH="path to the vae with .saftensor extension"
$env:OUTPUT_DIR= "path to the out put dir"
$env:MODEL_NAME=" path to the base model"
Normal that should work but if i run it i get my system can not found the path
env normal works for me on windows same as set in windows  because windows has no export command flag

**Comment by asomoza** [comment]
Hi, this probably means that the path you're using is not correct, also this will only work with powershell, you can test if the env variable is set correctly with this:

```
$env:INSTANCE_DIR = "C:\Path\To\The\Folder With The Images"
echo $env:INSTANCE_DIR
```

it should output: `C:\Path\To\The\Folder With The Images`.

Alternatively you can just skip the use of env variables an use the path directly in the script like this:

```
accelerate launch train_dreambooth_lora_sdxl.py --pretrained_model_name_or_path="C:\Path\sdxl_model" ...
```
